monitoring/v3/src/test/java/ListResourcesTest.java

```diff
@@ -34,7 +35,7 @@
   /**
    * The project ID of the project created for the integration tests.
    */
-  public static final String TEST_PROJECT_ID = "cloud-monitoring-dev";
+  public static final String TEST_PROJECT_ID = "cloud-samples-tests";
```

Comment:
Thanks for the heads up.

I was able to enable the API from my corp account, but it still might require some whitelisting. I get the following error when I run it:

```
$ GOOGLE_APPLICATION_CREDENTIALS=~/src/java-docs-samples/service-account.json mvn verify -DskipTests=false
...
Running ListResourcesTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.133 sec - in ListResourcesTest
Running CreateCustomMetricTest
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.234 sec <<< FAILURE! - in CreateCustomMetricTest
testValueRead(CreateCustomMetricTest)  Time elapsed: 0.234 sec  <<< ERROR!
com.google.api.client.googleapis.json.GoogleJsonResponseException: 
400 Bad Request
{
  "code" : 400,
  "errors" : [ {
    "domain" : "global",
    "message" : "The project 'cloud-samples-tests' is not in the requisite Stackdriver service tier.",
    "reason" : "badRequest"
  } ],
  "message" : "The project 'cloud-samples-tests' is not in the requisite Stackdriver service tier.",
  "status" : "INVALID_ARGUMENT"
}
...
```

I sent a request to the Cloud Monitoring team. I'll mail another PR to enable the tests when it does get whitelisted.